Owncloud iOs Client/Tabs/FileTab/FilesViewController.m

```diff
@@ -2316,6 +2316,7 @@ - (void)showSortingOptions{
  */
 - (void) updateActiveUserSortingChoiceTo: (enumSortingType)sortingChoice{
     [ManageUsersDB updateSortingWayTo:sortingChoice byUserDto:[ManageUsersDB getActiveUser]];
+    APP_DELEGATE.activeUser = [ManageUsersDB getActiveUser];
```

Comment:
We could update the sorting_type property directly instead of update all the user object and avoid extra request to the DB